qemu/tests/cfg/cpu_device_hotplug_maximum.cfg

```diff
@@ -0,0 +1,38 @@
+- cpu_device_hotplug_maximum:
+    virt_test_type = qemu
+    type = cpu_device_hotplug_maximum
+    no ovmf
+    no RHEL.6
+    only Linux
```

Comment:
We need test windows guest as well,  there is a bug only reproducible with windows guest.

Reply:
Could you help to provide CPU limit of each windows version, I don't know much about them.

Reply:
Seems they don't have a hard limitation on total cpu number, but the maximum socket number is 64.

Reply:
I see that the maximum number of CPU for some windows is 128, so I consider limiting the number of windows CPU to 128. Already updated.